apps/portals/e2e/explore.spec.ts

```diff
@@ -108,6 +113,70 @@ const getAvailableFacetsDiv = (page: Page) => {
     .locator('> div')
     .filter({ hasText: 'Available Facets' })
 }
+const toggleChartVisibilityAndVerifyCount = async (page: Page) => {
+  const chartsPanel = page.locator('.FacetNav')
+
+  const allChartCells = chartsPanel.locator('.FacetNavPanel')
+  const allChartsCount = await allChartCells.count()
+
+  const chartCells = chartsPanel.locator('.FacetNavPanel:visible')
+  const initialCount = await chartCells.count()
```

Comment:
Consider using `getByRole` if possible. I think we explicitly have the `"graphics-document"` role on these visualizations. I am not sure if `getByRole` works with hidden nodes